assets/src/edit-story/utils/getMediaBaseColor.js

```diff
@@ -56,11 +57,10 @@ function setOrCreateImage(src, onLoad) {
   }
   imgNode.innerHTML = '';
   const img = document.createElement('img');
-  img.src = src;
+  img.onload = onLoad;
+  img.crossOrigin = type === 'gif' ? undefined : 'anonymous';
```

Comment:
From chat: the weird behavior is due to `onLoad` never being called, so instead of toggling behavior based on `type` (a bit hacky), we can (1) always set `crossOrigin to "anonymous"` and (2) make sure `onLoad` is called when the image errors (`onerror`).

Reply:
@embarks To help unblock the release tomorrow (and cause I was already testing it), took a stab at this in 9d7710f. Hope you don't mind! :p

Reply:
Thank you :)